plugin/trino-hive/src/main/java/io/trino/plugin/hive/procedure/DropStatsProcedure.java

```diff
@@ -125,6 +125,7 @@ private void doDropStats(ConnectorSession session, ConnectorAccessControl access
                     schema,
                     table,
                     makePartName(partitionColumns, values),
+                    NO_ACID_TRANSACTION,
```

Comment:
Do we have a test for drop_stats and transactional table?

i guess it doesn't work, so please add `// TODO (issue ink) some text` here

(same below)

Reply:
Added, #7189